### PR TITLE
[HDFS-460] Add kerberos docs for HDFS

### PR DESCRIPTION
**Pull request by NimaVaziri**

### CODE REVIEW

File: frameworks/hdfs/docs/kerberos.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+}
+```
+Note the specification of the secret name as created in the previous step.
+
+The kerberized Apache HDFS service is then deployed by running:
+```bash
+$ dcos package install hdfs --options=kerberos-options.json
+```
+
+Note: The utility tool `kinit` needs to be installed on the agents and be accessible from within the tasks' containers in order for the system to deploy correctly.
```
Comment (benclarkwood) [inline frameworks/hdfs/docs/kerberos.md]: I'd put this at the top.

File: frameworks/hdfs/docs/kerberos.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+hdfs/journal-1-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
+HTTP/journal-1-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
+hdfs/journal-2-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
+HTTP/journal-2-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
+hdfs/data-0-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
+HTTP/data-0-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
+hdfs/data-1-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
+HTTP/data-1-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
+hdfs/data-2-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
+HTTP/data-2-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
```
Comment (benclarkwood) [inline frameworks/hdfs/docs/kerberos.md]: I'd put a `... up to N data nodes`

Comment (NimaVaziri) [reply frameworks/hdfs/docs/kerberos.md]: 👍

File: frameworks/hdfs/docs/kerberos.md (hunk 1 above)
Comment (NimaVaziri) [reply frameworks/hdfs/docs/kerberos.md]: Done

File: frameworks/hdfs/docs/kerberos.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+HTTP/hdfs.marathon.autoip.dcos.thisdcos.directory@LOCAL
+```
+This cluster has 3 data nodes, though the correct number of principles should be added up to N data nodes.
```
Comment (benclarkwood) [inline frameworks/hdfs/docs/kerberos.md]: s/principles/principals

Comment (NimaVaziri) [reply frameworks/hdfs/docs/kerberos.md]: ...right

**Pull request merged**